core/cache/machine.go

```diff
@@ -7,6 +7,7 @@ import (
 	"sync"
 
 	"github.com/juju/errors"
+	"github.com/juju/juju/core/instance"
```

Comment:
wrong section 'juju/juju' (this project) is its own section vs external non-stdlib section.

Reply:
I really need to get my stanza checker working as a pre-push for me :disappointed:

Reply:
I've got it working now - https://youtu.be/EguALVNLwjM